services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/settings.py

```diff
@@ -281,7 +281,7 @@ def _inject_destination_container(
         item: SimcoreServiceSettingLabelEntry,
     ) -> SimcoreServiceSettingLabelEntry:
         # pylint: disable=protected-access
-        item._destination_containers = destination_containers
+        item._destination_containers = destination_containers  # noqa: SLF001
```

Comment:
instead of accessing private members, why you do not create a method in SimcoreServiceSettingsLabelEntry? this would be cleaner.

Reply:
This does not work. If I create a getter and setter pedantic treats it as a property. It's a limitation, a private attribute must start with `_`

Reply:
well but you can still define methods in there

Reply:
🤦